docs/run-conformance-tests.md

```diff
@@ -0,0 +1,29 @@
+# Conformance tests
+
+This document enumerates the way steps to run conformance tests for various platforms supported by Lokomotive.
```

Comment:
We should have a reference to quick start document or configuration example, to have simple instruction how to create conforming cluster.

Reply:
done.